bn-apps/billing/billing-contracts-and-states/src/main/kotlin/com/r3/businessnetworks/billing/states/Schemas.kt

```diff
@@ -21,6 +21,21 @@ object BillingStateSchemaV1 : MappedSchema(schemaFamily = BillingState::class.ja
             var status : BillingStateStatus) : PersistentState()
 }
 
+@CordaSerializable
```

Comment:
Instead of creating a new schema would be better to define a migration for the current schema. Please see here: https://docs.corda.r3.com/database-management.html

Reply:
Yeh, I wondered about that.  We haven't had to do that before so thanks for pointing that out to me.